src/xlsx.js

```diff
@@ -21,27 +21,28 @@ export class Workbook {
   }
 }
 
-function extract(sheet, {range, headers = false} = {}) {
+function extract(sheet, {range, headers} = {}) {
   let [[c0, r0], [c1, r1]] = parseRange(range, sheet);
-  const headerRow = headers && sheet._rows[r0++];
+  const headerRow = headers ? sheet._rows[r0++] : null;
   let names = new Set(["#"]);
   for (let n = c0; n <= c1; n++) {
-    let name = (headerRow ? valueOf(headerRow._cells[n]) : null) || toColumn(n);
+    let name = (headerRow ? valueOf(headerRow._cells[n]) : null) || toColumn(n); // should a falsey header value be ignored? or just nullish?
```

Comment:
I think falsey, and specifically I'm thinking of ""¹ because I'm pretty sure with editing and re-editing someone could see a cell as empty in Excel, but the underlying value would be "". And with that, I could see wanting to maybe also ignore all-whitespace values here too.


¹ correct me if I'm wrong that that's not falsey.

Reply:
I was thinking of the value `false`. I agree that if it’s the empty string (which is also falsey) we want to replace that with a non-empty name. So, shouldn’t we coerce to a string here? 

```suggestion
    const value = headerRow ? valueOf(headerRow._cells[n]) : null;
    let name = value && (value += "") || toColumn(n);
```